README.md

```diff
@@ -3,16 +3,26 @@
 ![](https://img.shields.io/badge/TicTacToe-purple)
 ![](https://img.shields.io/badge/Ruby-red)
 
-# Building The Tic-Tac-Toe Game With Ruby
-How The Game Works
+# Steps for Playing Tic-Tac-Toe game
+
 - Two players are involved in the game
-- They start with an empty nine spaces in the board
-- Players will take the turn to put a marker on the board
-- The first player starts by selecting an empty slot between 1-9 on a 3 x 3 grid
-- The selected slot will be marked as 'X.' 
-- In the next turn, the second player will select another space, and the      cell will be marked with 'O'
-- The game is won when one player achieves three markers in a row - horizontally, vertically, or diagonally
-- The program will respond in conditions when the position is taken or when the player selects a number out of the given range or no winner. 
+- Each player has to enter their names one by one (Note!  You can not leave your name empty)
+- Playing borad <br>
+
+  " # | # | # " <br>
+  "-----------" <br>
+  " # | # | # " <br>
+  "-----------" <br>
+  " # | # | # " <br>
+  
+- The board will be displayed on the screen and the first player will be asked for selecting an empty cell on the board by selecting 1 – 9 number (Note: You cannot enter the number above or below the given range or select occupied space)
```

Comment:
... for <s>selecting</s> <b>select</b> an empty cell on the board by <s>selecting</s> <b>picking/choosing one number between 1 - 9 </b>.